app/models/speaker_email_template.rb

```diff
@@ -0,0 +1,5 @@
+class SpeakerEmailTemplate
```

Comment:
I'll move it to wherever it belongs, just let me know. I updated the PR. hopefully it's more what you want.